src/main/java/minicraft/core/Renderer.java

```diff
@@ -66,20 +67,16 @@ public static SpriteSheet[] loadDefaultSpriteSheets() {
 			// These set the sprites to be used.
 			itemSheet = new SpriteSheet(ImageIO.read(Objects.requireNonNull(Game.class.getResourceAsStream("/resources/textures/items.png"))));
 			tileSheet = new SpriteSheet(ImageIO.read(Objects.requireNonNull(Game.class.getResourceAsStream("/resources/textures/tiles.png"))));
-			entitySheet = new SpriteSheet(ImageIO.read(Objects.requireNonNull(Game.class.getResourceAsStream("/resources/textures/entities.png"))));
+			entitySheet = new SpriteSheet(ImageIO.read(Objects.requireNonNull(Game.class.getResourceAsStream("/resurces/textures/entities.png"))));
```

Comment:
Ah, you forgot to fix this after testing.

Reply:
Ah.